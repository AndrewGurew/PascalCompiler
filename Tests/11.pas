a = array[1001..12123] of integer;