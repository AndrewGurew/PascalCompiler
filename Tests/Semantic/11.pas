procedure bar(a:double; c:integer);
begin
end
Begin
bar(1,1.1);
end.