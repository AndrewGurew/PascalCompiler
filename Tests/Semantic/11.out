(6, 2) - expected Integer not Double